docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+|                     |                      | Corresponding author status | corresponding_author_status  | BOOLEAN                                 |  --  |   --    |                     OPTIONAL                      | N/A                                   | Indicating whether an author is the corresponding author (YES or NO)                                                                                                                                     |
+|                     |                      | Email                       | email                        | STRING                                  |  --  |   --    | MUST or OPTIONAL<sup>**[1](#author_email)**</sup> | N/A                                   | Email address for each author                                                                                                                                                                            |
+|                     |                      | Affiliation name            | affiliation_name             | STRING                                  |  --  |   --    |                    RECOMMENDED                    | ROR                                   | Name of the institution an author is affiliated with. Sometimes, one author may have multiple affiliations                                                                                               |
+|                     |                      | Affiliation address         | affiliation_address          | STRING                                  |  --  |   --    |                     OPTIONAL                      | N/A                                   | Address of the institution an author is affiliated with                                                                                                                                                  |
+|                     |                      | Affiliation identifier      | affiliation_identifier       | STRING                                  |  --  |   --    |                    RECOMMENDED                    | ROR                                   | A unique identifier assigned to the affiliated institution by The Research Organization Registry (ROR)                                                                                                   |
+|                     |                      | Author list order           | order                        | INTEGER                                 |  --  |   --    |                       MUST                        | N/A                                   | The order in which the author appears in the publication                                                                                                                                                 |
```

Comment:
The 'Author list order' is not actually a field in the JSON files, it is explictely given by the order in the list (JSON arrays are ordered).

Reply:
Yeah, I think we shouldn't include the field in the metadata spec and just say that the ordering is significant.